src/ucp/api/ucp.h

```diff
@@ -259,14 +259,41 @@ enum ucp_ep_params_flags_field {
 };
 
 
+/**
+ * @ingroup UCP_ENDPOINT
+ * @brief Close UCP endpoint modes.
+ *
+ * The enumeration is used to specify the behavior of @ref ucp_ep_close_nb.
+ */
+enum ucp_ep_close_flags_t {
```

Comment:
typedef enum {
} ucp_ep_close_flags_t ;

Reply:
hmmm, I used same definition as for original/other enums...
ok, fixed